### ISSUE TITLE
RawRecord is showing extra spaces

### ISSUE THREAD

**Issue by rgiosa** [JoshClose/CsvHelper#1280]
Hi,

I am having problems using `csv.Context.RawRecord`. I don't know if it is a bug. I have a simple CSV file:

```
Code;FieldA;FieldB
1;Something;Code 1
2;Another;Code 2
```

When I inspect `csv.Context.RawRecord` the value is (notice the extra space in the last field):
<pre>
"1;Something;Code  1\r\n"
</pre>

This is how I am loading the file:
```csharp
using (var reader = new StreamReader(originalStream, Encoding.Default))
{
   var csv = new CsvReader(reader);
   csv.Configuration.Encoding = Encoding.Default;
   csv.Configuration.Delimiter = ";";
   csv.Configuration.TrimOptions = CsvHelper.Configuration.TrimOptions.Trim;
   csv.Configuration.BadDataFound = null;

   csv.Read();

   do
   {
      var row = csv.Context.RawRecord;
      //...
   } while (csv.Read());
}
```

**CsvHelper Version:** 12.1.2

**Comment by JoshClose** [comment]
Looks like a bug to me.

**Comment by wayneseguin** [comment]
I am seeing this exact same behavior. To add a small detail, EVERY space in the record is doubled in the Context.RawRecord.

**Comment by Kaiyaq** [comment]
Is there a workaround this issue?

**Comment by Dehalion** [comment]
I have the same problem, every space is doubled in `RawRecord`.
Looks like a bug to me too.

**Comment by Dehalion** [comment]
The problem is when you use `CsvHelper.Configuration.TrimOptions.Trim`.
All spaces in fields not quoted will get doubled if you use this option.

**Comment by JoshClose** [comment]
This is fixed and will be in release >= 12.2.1. Thanks to @Dehalion for the pull request.
